src/lib/temple/back/vault.ts

```diff
@@ -113,6 +112,12 @@ export class Vault {
     });
   }
 
+  static async clearStorage() {
```

Comment:
Move this method from Vault to separate method.

Reply:
I think a new module like "lib/temple/reset" would be a great place for this stuff.